api/aqua-api-example/index.js

```diff
@@ -0,0 +1,30 @@
+import {
+    Aqua,
+    Call,
+    Path,
+} from "@fluencelabs/aqua-api/aqua-api.js";
+import { FluencePeer } from "@fluencelabs/fluence";
+import { callFunctionImpl } from "@fluencelabs/fluence/dist/internal/compilerSupport/v3impl/callFunction.js";
+
+const aquaPath = new Path("test.aqua")
+// write function that we want to call and arguments
+const args = {num: 42}
+const call = new Call("getNumber(num)", args, aquaPath)
+
+// compile call
+const compilationResult = await Aqua.compile(call, [])
+// get function definition and AIR script
+const {funcDef, script} = compilationResult.functionCall
+
+// initialize fluence client
+const fluence = new FluencePeer();
+await fluence.start({})
+
+// make a call with arguments
+const result = await callFunctionImpl(funcDef, script, {}, fluence, args)
```

Comment:
I think that instead of creating FluencePeer, it would be nicer to print out `compilationResult` and explain what it means.

right now, it's all a black box that's passed between `Aqua.compile` and `callFunctionImpl`.